tsconfig.json

```diff
@@ -1,5 +1,4 @@
 {
-  "extends": "@tsconfig/next/tsconfig.json",
```

Comment:
For some reason i am getting error while extending this. says "not found" !!

Reply:
![CleanShot 2025-02-07 at 10  29 32](https://github.com/user-attachments/assets/9c356fdb-e44d-4ce8-a610-c0a24cd8dcfd)

Also getting this error when running `pnpm run validate` 🤔  any idea guys?

Reply:
This only occurs when `pnpm run check-types and pnpm run build` are executed together; it doesn't happen when run separately.

Reply:
The problem occurred because check-types and build were running concurrently. Since build removes and regenerates the contents of the .next directory,` tsc` (running in the background) was unable to properly reference the files, leading to the error.

To fix it: if you check the `validate` command i made a small change:
```
pnpm run prettier & pnpm run lint & pnpm run check-types && pnpm run build
```
I used `&&` instead of `&` to ensure that the second command (build) runs only if the first command (check-types) succeeds. Let me know if you have any questions!